### PR TITLE
Snapshot ingestion procedure for CSI Unity Driver

### PR DESCRIPTION
**Pull request by HarishH-DELL**
# Description
Snapshot ingestion procedure for CSI Unity Driver.

# GitHub Issues
List the GitHub issues impacted by this PR:

| GitHub Issue # |
| -------------- |
|https://github.com/dell/csm/issues/1206|

# Checklist:

- [X] Have you run a grammar and spell checks against your submission?
- [ ] Have you tested the changes locally?
- [ ] Have you tested whether the hyperlinks are working properly?
- [X] Did you add the examples wherever applicable?
- [ ] Have you added high-resolution images?

### CODE REVIEW

File: content/docs/csidriver/features/unity.md [hunk 1]
```diff
@@ -211,6 +211,63 @@ spec:
...
+    deletionPolicy: Delete
+    driver: csi-unity.dellemc.com
+    volumeSnapshotClassName: unity-snapclass
+    source:
+      snapshotHandle: snap1-<protocol>-<array_id>-<snapshot_id>
+    volumeSnapshotRef:
+      name: snap1
+      namespace: unity
+  ```
+>Exampple snapshot handle format: snap1-FC-apm00123456789-3865491234567 .
```
Comment (gallacher) [inline content/docs/csidriver/features/unity.md:236]: Typo

File: content/docs/csidriver/features/unity.md [hunk 2]
```diff
@@ -211,6 +211,63 @@ spec:
...
+ metadata:
+   name: snap1
+   namespace: unity
+ spec:
+   volumeSnapshotClassName: unity-snapclass
+   source:
+     volumeSnapshotContentName: snap1-content
+ ```
+
+4. Create a PersistentVolumeClaim explained below
```
Comment (gallacher) [inline content/docs/csidriver/features/unity.md:252]: Step 3 has a space between the step and the start of the yaml but there isn't one here.  Does that impact anything?

File: content/docs/csidriver/features/unity.md [hunk 3]
```diff
@@ -211,6 +211,63 @@ spec:
 
 >The Kubernetes Volume Expansion feature can only be used to increase the size of a volume. It cannot be used to shrink a volume.
 
+
+## Snapshot ingestion procedure
```
Comment (gallacher) [inline content/docs/csidriver/features/unity.md:215]: Please provide some text describing the snapshot ingestion and what this section will outline.

File: content/docs/csidriver/features/unity.md (hunk 2 above)
Comment (HarishH-DELL) [reply content/docs/csidriver/features/unity.md:255]: fixed the space issue

File: content/docs/csidriver/features/unity.md (hunk 3 above)
Comment (HarishH-DELL) [reply content/docs/csidriver/features/unity.md:215]: added description

**Review by rajendraindukuri — APPROVED** [review]

**Review by rajkumar-palani — APPROVED** [review]
lgtm

**Pull request merged**